module-types/Cisco/N7K-AC-6KW.yaml

```diff
@@ -0,0 +1,8 @@
+---
+manufacturer: Cisco
+model: N7K-AC-6.0KW
+part_number: N7K-AC-6.0KW
+comments: '[Cisco Nexus 7000 Series AC Power Supply Modules Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/nexus-7000-series-switches/Data_Sheet_C78-437761.html)'
+weight:
+  - value: 18
+    unit: lb
```

Comment:
Looks like this is a power supply, yes? If so, please add the applicable `power-ports`